src/scripts/i18n/inline_and_diff.py

```
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)